approve.py

```diff
@@ -0,0 +1,33 @@
+from config import config
+# return search id from given target name
+
+
+def get_search_id(saved_search_list, target_name):
+    for search in saved_search_list:
+        if search["Name"].lower() == target_name:
+            return search["Id"]
+    return None
+
+# method to approve contact by Id
+
+
+def approve_membership(session, contact_id):
+    temp = 'ApprovePendingMembership?contactId='+str(contact_id)
```

Comment:
Let's use format strings instead of concatenation like this, since we use them everywhere else.
https://docs.python.org/3/library/string.html#format-string-syntax

Reply:
Sorry, this is the wrong link.
https://docs.python.org/3/reference/lexical_analysis.html#f-strings

Reply:
```python
temp = f'ApprovePendingMembership?contactId={contact_id}'
```

Reply:
Thank you for the example.